students/AndrewMiotke/session03/list_lab.py

```diff
@@ -34,14 +34,15 @@
 print(fruit_list)
 
 # Series Three
-while True:
-    for favorite_fruit in fruit_list:
-        users_favorite_fruit = input(f"do you like {favorite_fruit.lower()}, yes or no? ")
-        if users_favorite_fruit == "no":
-            fruit_list.remove(favorite_fruit)
-            print(fruit_list)
-        elif users_favorite_fruit != "yes" or "no":
-            print(fruit_list)
+for favorite_fruit in fruit_list:
```

Comment:
You will want to be careful iterating here. If you run this and say no, you will delete the fruit the it will "skip" the next fruit in line. Try this out and see what happens